### ISSUE TITLE
On blur not working

### ISSUE THREAD

**Issue by be-codified** [react-bootstrap/react-bootstrap#1917]
Thank you for the great pack.

I can not figure out why onBlur is not working, although `onClick` is.

    let handleChangeBind = this.handleChange.bind(this);
    
    <FormGroup
        controlId="firstName"
        validationState={this.getValidationState()}
    >
        First name
        <FormControl
            type="text"
            value={this.state.value}
            onBlur={handleChangeBind}
        />
    </FormGroup>

In this case I even can not type in input field. Any ideas?

**Comment by jquense** [comment]
You are using react controlled inputs incorrectly. The input will always be the value you give it in `value` since you aren't updating that value as you type it does not update at all. https://facebook.github.io/react/docs/forms.html

You need to use defaultValue or onChange

**Comment by be-codified** [comment]
I am sorry, I don't understand your answer. Is there a way to trigger method when users leave input field?

I don't want to have 'live' validation when user is typing.

**Comment by jquense** [comment]
you can use onBlur to trigger something when the user leaves the field, but you _cannot_ adjust the value of the input onClur, you have to do `setState({ value })` onChange, or use an uncontrolled input. Read the link I posted for more info
